### ISSUE TITLE
Tags de capítulo

### ISSUE THREAD

**Issue by cescoferraro** [livrospringmvc/lojacasadocodigo#2]
Acabei de comprar o livro e acabei caindo aqui nesse repósitório. Tem como tagear os commits em função dos capítulos, acho que fica mais fácil acompanhar. Alguns livros da O'Reilley que lí usam esse método e funciona bastante. Valeu

**Comment by khouri** [comment]
Concordo com o colega cescoferraro, por capítulo facilita o aprendizado

**Comment by arturcorvino** [comment]
É, também acho interessante. Comprei o livro e estou fazendo isso no meu repositório. Porém tive alguns problemas em um dos capitulos.

**Comment by PedroPK** [comment]
Assino embaixo. O repositório com tudo pronto fica desconectado do livro.

Ou como alternativa, como já vi em repositórios de outros livros da Casa do Código, cada capítulo ter um repositório separado.
